[
    {
        "code": "url_not_provided",
        "message": "`url` is not provided."
    },
    {
        "code": "invalid_url",
        "message": "`url` must start with `https://`."
    },
    {
        "code": "invalid_cloud_scenario_url",
        "message": "The specified url is not valid. To authenticate with your Cloud Pak for Data installed software, add `\"instance_id\": \"openshift\"` to your credentials. To authenticate with your Cloud Pak for Data as a Service account, see https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/ml-authentication.html ."
    },
    {
        "code": "apikey_not_provided",
        "message": "'`apikey` for IAM token is not provided in credentials for the client'"
    },
    {
        "code": "version_not_provided",
        "message": "Missing version in credentials for Cloud Pak for Data. Add key `version` to your credentials with a supported value, referring to your software version: {0}"
    },
    {
        "code": "no_port_number_for_wml_local",
        "message": "You must specify a port number as part of the url for wml_local."
    },
    {
        "code": "invalid_version",
        "message": "Invalid value for 'version' provided in wml_credentials. Update the wml_credentials and try again. Supported value for version field are: {0}"
    },
    {
        "code": "instance_id_in_cloud_scenario",
        "message": "Specified credentials are invalid. Do not specify `instance_id` for Cloud Pak for Data as a Service. Update your wml_credentials and try again. See https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/ml-authentication.html ."
    },
    {
        "code": "invalid_instance_id",
        "message": "Invalid instance_id for Cloud Pak for Data. Use `\"instance_id\": \"openshift\"` in your credentials. To authenticate with a different offering, refer to the product documentation for authentication details."
    },
    {
        "code": "instance_id_is_not_required_for_v2_plans",
        "message": "You do not have to specify 'instance_id' for this instance of Watson Machine Learning. It will be picked up from space when `client.set.default_space(space_id)` is called."
    },
    {
        "code": "creating_assets_using_v4_beta_flow_is_deprecated",
        "message": "Support for the V4 beta version of the Watson Machine Learning APIs is discontinued. Refer to the documentation at https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/wml-ai.html for the migration process to be able to access new features."
    },
    {
        "code": "instance_id_not_found",
        "message": "`instance_id` not found. You do not have to specify `instance_id` for this instance of Watson Machine Learning. It will be picked up from space when `client.set.default_space(space_id)` is called."
    },
    {
        "code": "not_authorized_to_access_the_instance_id",
        "message": "You are not authorized to access the instance ID for this Watson Machine Learning instance. Note that it will be picked up from space when `client.set.default_space(space_id)` is called. Remove `instance_id` from credentials and try again."
    },
    {
        "code": "failed_to_get_instance_details",
        "message": "Failed to get details for the instance ID for this Watson Machine Learning instance. It will be picked up from space when `client.set.default_space(space_id)` is called. Remove `instance_id` from credentials and try again.\nError: {0}"
    },
    {
        "code": "provided_credentials_are_invalid",
        "message": "The specified credentials are invalid. `instance_id` and `version` keys provided are not correct. Update your wml_credentials and try again."
    },
    {
        "code": "client_is_not_supported_in_this_release",
        "message": "This client is not supported in this release. Refer to the documentation at https://ibm.github.io/watson-machine-learning-sdk to know what releases are supported"
    },
    {
        "code": "python_3_6_framework_is_deprecated",
        "message": "Python 3.6 framework is no longer supported. Use Python 3.7 instead. For details, see https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/pm_service_supported_frameworks.html"
    },
    {
        "code": "python_3_6_framework_is_deprecated_cp4d",
        "message": "The Python 3.6 framework is no longer supported. We recommend you migrate your assets and use a Python 3.7 framework instead. For details, see https://www.ibm.com/support/knowledgecenter/SSQNUZ_current/wsj/analyze-data/pm_service_supported_frameworks.html"
    },
    {
        "code": "python_3_7_3_8_framework_are_deprecated",
        "message": "Python 3.7 and 3.8 frameworks are deprecated and will be removed in a future release. Use Python 3.9 framework instead."
    },
    {
        "code": "client_successfully_initialized",
        "message": "Client successfully initialized"
    },
    {
        "code": "it_is_mandatory_to_set_the_space_project_id",
        "message": "You must set the space/project id. Use `client.set.default_space(<SPACE_UID>)`/`client.set.default_project(<PROJECT_UID>)` to proceed."
    },
    {
        "code": "it_is_mandatory_to_set_the_space_id",
        "message": "You must set the space id. Use `client.set.default_space(<SPACE_UID>)` to proceed."
    },
    {
        "code": "it_is_mandatory_to_set_the_project_id",
        "message": "You must set the project id. Use `client.set.default_project(<PROJECT_UID>)` to proceed."
    },
    {
        "code": "apikey_value_cannot_be_empty",
        "message": "`apikey` value cannot be ''. Pass a valid apikey for IAM token."
    },
    {
        "code": "token_value_cannot_be_empty",
        "message": "`token` value cannot be ''. Pass a valid token for IAM token."
    },
    {
        "code": "module_mimetypes_not_found",
        "message": "Module mimetypes not found."
    },
    {
        "code": "creating_data_asset",
        "message": "Creating data asset..."
    },
    {
        "code": "failed_while_creating_a_data_asset",
        "message": "Failed while creating a data asset. Try again."
    },
    {
        "code": "success",
        "message": "SUCCESS"
    },
    {
        "code": "failure_during_downloading_data_asset",
        "message": "Failure during downloading data asset."
    },
    {
        "code": "successfully_saved_data_asset_content_to_file",
        "message": "Successfully saved data asset content to file: '{0}'"
    },
    {
        "code": "saving_data_asset_to_local_file_failed",
        "message": "Saving data asset with artifact_url: '{0}'  to local file failed."
    },
    {
        "code": "download_api_not_supported_for_this_connection_type",
        "message": "Download API is not supported for Data Assets created using this connection type. It is supported only for Cloud storage data source type connections"
    },
    {
        "code": "failed_to_read_response_from_down_stream_service",
        "message": "Failed to read Response from down-stream service: {0}"
    },
    {
        "code": "creating_connections",
        "message": "Creating connections..."
    },
    {
        "code": "failed_while_creating_connections",
        "message": "Failed while creating a Connections. Try again."
    },
    {
        "code": "obm_deprecation_message_cpd",
        "message": "AutoAI experiments with joined data deprecated\nUser action required: The AutoAI feature for joining multiple data sources into a single training data set is deprecated. Support will be removed in a future refresh of Cloud Pak for Data. Use a data preparation tool such as Data Refinery or DataStage to join data, then use the resulting data set for training an experiment. Redeploy the resulting model."
    },
    {
        "code": "obm_removal_message_cpd",
        "message": "AutoAI experiments with joined data not supported.\nUser action required: The AutoAI feature for joining multiple data sources into a single training data set is not supported in Cloud Pak for Data 4.6 release and later. Use a data preparation tool such as Data Refinery or DataStage to join data, then use the resulting data set for training an experiment. Redeploy the resulting model."
    },
    {
        "code": "obm_removal_message_cloud",
        "message": "AutoAI experiments with joined data not supported.\nUser action required: The AutoAI feature for joining multiple data sources into a single training data set is not supported in Cloud Pak for Data as a Service. Use a data preparation tool such as Data Refinery or DataStage to join data, then use the resulting data set for training an experiment. Redeploy the resulting model."
    }
]
